backend/inspirehep/disambiguation/tasks.py

```diff
@@ -70,13 +72,38 @@ def disambiguate_signatures(self, clusters):
     db.session.commit()
 
 
+def match_literature_author(author_data, matcher_config):
+    matched_records = [
+        matched_record["inner_hits"]["authors"]["hits"]["hits"]
+        for matched_record in match(author_data, matcher_config)
+    ]
+    matched_authors_refs = []
```

Comment:
you could use `get_value()` https://github.com/inspirehep/inspire-utils/blob/bf94a7ad05fac5925b625bfc2491c54b4cfb5044/inspire_utils/record.py#L33